index.html

```diff
@@ -12,50 +12,188 @@
-                    </li>
-                    <li class="icon-item">
-                        <a href="#"><img src="images/github.png" alt=""></a>
-                    </li>
-                    <li class="icon-item">
-                        <a href="#"><img src="images/angellist.png" alt=""></a>
-                    </li>
-                </ul>
-            
-            <!-- action button -->
-            <a href="" class="action-button">
-                <img src="images/Action.png" alt="action button">
-            </a>
-        </section>
+  <!-- Toolbar -->
+  <header class="header">
+      <nav class="header-nav flex">
+          <a href="#" class="logo">
+              <img src="images/Welcome.png" alt="welcome logo">
+          </a>
+          <a href="#" class="nav-menu">
+              <img src="images/Union.png" alt="hamburger menu">
+          </a>
+      </nav> 
+  </header>
+
+  <!-- Headline Section -->
+  <section class="headline flex">
+      <h1 class="headline-title">Hi there. I'm Mohamed<br><span>I'm a software developer</span></h1>   
+        <p class="headline-subtitle">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+  
+          <!-- social media icons -->
+          <!-- <a href="#" class="icons">
+              <img class="icon-img" src="images/social-media-icons.png" alt="Social Media icons">
+          </a> -->
+          <ul class="icons flex">
+              <li class="icon-item">
+                  <a href="#"><img src="images/twitter.png" alt=""></a>
+              </li>
+              <li class="icon-item">
+                  <a href="#"><img src="images/Linkedin-icon.png" alt=""></a>
+              </li>
+              <li class="icon-item">
+                  <a href="#"><img src="images/medium.png" alt=""></a>
+              </li>
+              <li class="icon-item">
+                  <a href="#"><img src="images/github.png" alt=""></a>
+              </li>
+              <li class="icon-item">
+                  <a href="#"><img src="images/angellist.png" alt=""></a>
+              </li>
+          </ul>
+      
+      <!-- action button -->
+      <a href="" class="action-button">
+          <img src="images/Action.png" alt="action button">
+      </a>
+  </section>
+
+  <!-- work section -->
+  <section class="work-container">
+    <div class="work-heading">
+      <h2>My Recent Works</h2>
+      <div class="line"></div>
+    </div>
+    <div class="grid-container">
+      <div class="grid-card">
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="tags">
+          <li class="tag-item">Ruby on rails</li>
+          <li class="tag-item">CSS</li>
+          <li class="tag-item">JavaScript</li>
+          <li class="tag-item">HTML</li>
```

Comment:
- [ ] Please ensure that the `color` and the `font-weight` of the technologies tag match those specified in the design
- [ ] Your technologies list is skewed to the right instead of centered. Please fix this
### Your Design
![image](https://user-images.githubusercontent.com/46839548/185723756-8cc1be77-58ce-4071-a83e-391c70836559.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/185723740-2e019839-2126-4668-b352-2b781633169f.png)